### PR TITLE
Fixed inspector not updating anchor

### LINKED ISSUE #49004: Anchor in inspector does not reset when set via Layout menu
**Godot version:**
3.3.0

**OS/device including version:**
MacOS Catalina

**Issue description:**
When I set layout via the layout menu, then hit cmd-z to undo, the old anchor values still display in the inspector.

**Steps to reproduce:**
1. Make an HBoxContainer
2. Click Layout > Full Rect
3. Press Cmd-Z
4. Notice that the inspector still reads Anchor>Right 1 and Anchor>Bottom 1, even though they aren't 1 any more. 

**Minimal reproduction project:**
This reproduces in an empty project.

### PR DESCRIPTION
**Pull request by amoriqbal**
Fixes #49004
**Before:**
The change in preset-layout was taking place. But the respective changes in `anchor` were not being notified to the inspector.

**After:**
`_change_notify("anchor_<left/right/top/bottom>")` was added to the setter of `anchor` and `margin` properties.
This fixes the bug.

### CODE REVIEW

**Comment by amoriqbal** [comment]
Any updated on this? Did I request for the wrong branch?

**Comment by pycbouh** [comment]
> Any updated on this?

Hey there. Reviewing PRs takes time, it can be days or weeks before this is merged, so please be patient.

> Did I request for the wrong branch?

As a matter of fact, yes. All PRs must be done against the `master` branch first, and can only then be backported to `3.x`. If for some reason this problem doesn't exist in `master` and there is no reason to improve code there, you still would need to make your PR against `3.x`, not `3.3`.

**Comment by amoriqbal** [comment]
> you still would need to make your PR against 3.x

Much needed info. Thanks a lot.

File: scene/gui/control.cpp [hunk 1]
```diff
@@ -88,6 +88,11 @@ void Control::_edit_set_state(const Dictionary &p_state) {
 	data.margin[MARGIN_RIGHT] = margins[2];
 	data.margin[MARGIN_BOTTOM] = margins[3];
 	_size_changed();
+	update();
```
Comment (KoBeWi) [inline scene/gui/control.cpp:91]: The `update()` is not necessary here.

**Comment by amoriqbal** [comment]
I did not understand this. I force pushed the PR branch and the issue automatically got closed? It even got merged. But the issue #49004 is still open. This PR fixes that.

**Comment by KoBeWi** [comment]
It's not merged, you accidentally removed all changes.

**Comment by amoriqbal** [comment]
okay... sorry for the trouble

**Comment by amoriqbal** [comment]
Should I open another PR?

**Comment by akien-mga** [comment]
Superseded by #49167.

**Pull request closed without merging**